core/src/main/java/org/testcontainers/dockerclient/DockerClientConfigUtils.java

```diff
@@ -18,41 +18,42 @@
     public static final boolean IN_A_CONTAINER = new File("/.dockerenv").exists();
 
     @Getter(lazy = true)
-    private static final Optional<String> detectedDockerHostIp = Optional
-            .of(IN_A_CONTAINER)
-            .filter(it -> it)
-            .map(file -> DockerClientFactory.instance().runInsideDocker(
-                    cmd -> cmd.withCmd("sh", "-c", "ip route|awk '/default/ { print $3 }'"),
-                    (client, id) -> {
-                        try {
-                            LogToStringContainerCallback loggingCallback = new LogToStringContainerCallback();
-                            client.logContainerCmd(id).withStdOut(true)
-                                                      .withFollowStream(true)
-                                                      .exec(loggingCallback)
-                                                      .awaitStarted();
-                            loggingCallback.awaitCompletion(3, SECONDS);
-                            return loggingCallback.toString();
-                        } catch (Exception e) {
-                            log.warn("Can't parse the default gateway IP", e);
-                            return null;
-                        }
-                    }
-            ))
-            .map(StringUtils::trimToEmpty)
-            .filter(StringUtils::isNotBlank);
+    private static final Optional<String> defaultGateway = Optional
```

Comment:
I know that the indentation was incorrect, but could you please revert the change to the actual change? We will fix the indentation afterwards, but it will help to understand what was modified :)

Reply:
I think the whole change is as simple as `getDefaultGateway().orElse("localhost")` in `getDockerHostIpAddress`

Reply:
okay will fixed